virttest/asset.py

```diff
@@ -519,8 +522,12 @@ def download_file(asset_info, interactive=False, force=False):
                 if answer == 'y':
                     logging.info("Updating image to the latest available...")
                     while not file_ok:
-                        download.url_download_interactive(url, destination,
-                                                          title)
+                        try:
+                            download.url_download_interactive(url, destination,
+                                                              title)
+                        except Exception, e:
```

Comment:
Hi @balamuruhans 
I used the same pattern uses in line 481 of the same file:

            except Exception, e:
                logging.error("Failed to get SHA1 from file: %s", e)

I can remove from both and resend the PR.